crates/uv-configuration/src/project_build_backend.rs

```diff
@@ -5,16 +5,24 @@
 #[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
 pub enum ProjectBuildBackend {
     #[default]
+    #[serde(alias = "hatchling")]
+    #[value(alias = "hatchling")]
     /// Use [hatchling](https://pypi.org/project/hatchling) as the project build backend.
     Hatch,
     /// Use [flit-core](https://pypi.org/project/flit-core) as the project build backend.
+    #[serde(alias = "flit-core")]
+    #[value(alias = "flit-core")]
     Flit,
     /// Use [pdm-backend](https://pypi.org/project/pdm-backend) as the project build backend.
+    #[serde(alias = "pdm-backend")]
+    #[value(alias = "pdm-backend")]
     PDM,
     /// Use [setuptools](https://pypi.org/project/setuptools) as the project build backend.
     Setuptools,
     /// Use [maturin](https://pypi.org/project/maturin) as the project build backend.
     Maturin,
     /// Use [scikit-build-core](https://pypi.org/project/scikit-build-core) as the project build backend.
+    #[serde(alias = "scikit-build", alias = "scikit-build-core")]
+    #[value(alias = "scikit-build", alias = "scikit-build-core")]
```

Comment:
```suggestion
    #[serde(alias = "scikit-build-core")]
    #[value(alias = "scikit-build-core")]
```

I would omit `scikit-build` alias as its a different build backend than `scikit-build-core`.

Reply:
Oh gosh those are separate? 😭 need to fix that in the documentation. Thanks!